server/lib/util/filterEvaluator.js

```diff
@@ -0,0 +1,91 @@
+const _ = require('lodash')
+
+module.exports = function ExpressionParser () {
```

Comment:
This is actually really generic and could be published as its own npm module with docs that demonstrate usage.

Reply:
I think it might be a better idea to extract this.

BTW, have you searched for a lib that does the same thing?

Reply:
I tried to search for something like this but I couldn't find anything. What would you even call this? I don't even know how to describe it in 2-3 words so I found it very hard to search for.

I think it would be great to extract. We should be constantly looking for opportunities to extract and open source internal libs like this. Where should the repo go? under github.com/aerogear?